src/app/digital/views/info/index.ts

```diff
@@ -2,15 +2,16 @@ import {PropInfoRecord} from "core/models/PropInfo";
 
 import {DigitalObj} from "core/models/types/digital";
 
-import {DefaultComponentPropInfo, DefaultPortPropInfo, DefaultWirePropInfo} from "core/views/DefaultPropInfo";
+import {ConstantNumberPropInfo, DefaultComponentPropInfo, DefaultPortPropInfo, DefaultWirePropInfo} from "core/views/DefaultPropInfo";
 
 
 export const DigitalPropInfo: PropInfoRecord<DigitalObj> = {
     "DigitalPort": DefaultPortPropInfo,
     "DigitalWire": DefaultWirePropInfo,
     "DigitalNode": DefaultComponentPropInfo,
 
-    "Switch":  DefaultComponentPropInfo,
-    "LED":     DefaultComponentPropInfo,
-    "ANDGate": DefaultComponentPropInfo,
+    "Switch":         DefaultComponentPropInfo,
+    "LED":            DefaultComponentPropInfo,
+    "ANDGate":        DefaultComponentPropInfo,
+    "ConstantNumber": ConstantNumberPropInfo,
```

Comment:
Just hardcode the props directly on this line, no need to make them in `core/views/DefaultPropInfo`